lib/matplotlib/quiver.py

```diff
@@ -186,6 +186,10 @@
 
     %(PolyCollection_kwdoc)s
 
+Returns
+----------------
+The new `matplotlib.quiver.Quiver` object.
```

Comment:
```suggestion
-------
matplotlib.quiver.Quiver
```
See https://numpydoc.readthedocs.io/en/latest/format.html#sections on the `Returns` section.